### ISSUE TITLE
Number of iframes

### ISSUE THREAD

**Issue by AdrianTimpson** [BIADcore/BIADwiki#19]
Since the number of tables in BIAD changes, we need a better way of including all the table iframes in BIADwiki/structure page.
Several possibilities:
1. Weekly R auto script to email the administrator if the number of iframes doesn't match the number of tables, so administrator can manually update the html
2. Combine all html outputs into a single html in a single iframe in BIADwiki/structure
3. auto update the BIADwiki/structure page with additional iframes accordingly.

**Comment by simoncarrignon** [comment]
I think 2. would be the easiest way. Combining all table as html in a broad html page wouldn't be to hard (don't know with `gt` but for sure with `rmardkown/knitr` that should be easy. And redirect to the page. I started looking at updating the wiki page  (so downloading, and uploading it through the REST API) but tbh that look more painfull...

**Comment by AdrianTimpson** [comment]
Agreed! 1) A single combined.html. 2) Most of the content of tools can go in the .gitignore. BIADwiki.sh still needs to pull, but yes lets get rid of the push.
